lib/kubeclient.rb

```diff
@@ -13,6 +13,9 @@ module Kubeclient
   class Client < Common::Client
     attr_reader :api_endpoint
 
+    SERVICE_ACCOUNT_TOKEN_FILE =
```

Comment:
It's just a convenience so that users don't have to remember the service token path. It will only be used if passed in to the client constructor, not used by default.